games/Hangman.py

```diff
@@ -0,0 +1,116 @@
+import random
+
+# simple terminal hangman game with ASCII art
+def choose_random_word():
+    word_list = ["hangman", "python", "programming", "developer", "challenge", "knowledge"]
+    return random.choice(word_list)
+
+def display_word(word, guessed_letters):
+    display = ''
+    for letter in word:
+        if letter in guessed_letters:
+            display += letter
+        else:
+            display += '_'
+    return display
+
+def draw_hangman(attempts):
+    hangman_stages = [
+        """
+           ------
+           |    |
+           |
+           |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |    |
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |   /
+           |
+        """,
+        """
+           ------
+           |    |
+           |    O
+           |   /|\\
+           |   / \\
+           |
+        """
+    ]
+    
+    print(hangman_stages[attempts])
+
+def hangman():
+    word_to_guess = choose_random_word()
+    max_attempts = len(word_to_guess) + 2
```

Comment:
Major: this should be:

`max_attempts = 6`

Because we only have 7 different drawings, if we use the length of the word and word's length is for example 8, as soon as we guess wrong the 7th time we will get an IndexError as we do not have a hangman_stages of index 7.

Reply:
Thanks! I fixed the issue now.